doc/changelog.rst

```diff
@@ -3,8 +3,156 @@
 Changelog
 =========
 
+.. _changelog-v2-11-0:
+
+v2.11.0
+-------
+Highlights
+
+TODO: add highlights
+
+This release includes
+
+Backwards incompatible changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+TODO: add examples of how to deal with these changes
+
+-  Allow contiguous representation of extracted regions (:pull:`2230`) by :user:`rebeccaherman1`
+
+Deprecations
+~~~~~~~~~~~~
+
+-  Refactor regridding (:pull:`2231`) by :user:`schlunma`
+-  Allow deprecated regridding scheme `linear_extrapolate` in recipe checks (:pull:`2324`) by :user:`schlunma`
+-  Allow deprecated regridding scheme `unstructured_nearest` in recipe checks (:pull:`2336`) by :user:`schlunma`
+
+Bug fixes
+~~~~~~~~~
+
+-  Fixing missing height 2m coordinates in GFDL-CM4 and KIOST-ESM (:pull:`2294`) by :user:`Karen-A-Garcia`
```

Comment:
This should be in the 'Fixes for datasets' section

Reply:
Fixed here: 657b4598b

The `Fixes for datasets` label was on the PR.  Is it worth updating the script to have a second set of prioritised labels that are used in the groupings?

Reply:
Thanks! I wouldn't put too much effort in updating the script. The @ESMValGroup/technical-lead-development-team actually agreed to try out [towncrier](https://towncrier.readthedocs.io/en/latest/tutorial.html) instead some time ago because the current changelog is not great for users and difficult to populate for release managers who are not up-to-date with everything that happened in the project over the past few months.

Reply:
Oh great! Thanks for letting us know.